packages/replay/src/coreHandlers/handleDom.ts

```diff
@@ -27,12 +26,13 @@ export const handleDomListener: (replay: ReplayContainer) => (handlerData: DomHa
     }
 
     const isClick = handlerData.name === 'click';
-    const event = isClick && (handlerData.event as PointerEvent);
+    const event = isClick ? (handlerData.event as PointerEvent) : undefined;
     // Ignore clicks if ctrl/alt/meta/shift keys are held down as they alter behavior of clicks (e.g. open in new tab)
     if (
       isClick &&
       replay.clickDetector &&
       event &&
+      event.target &&
```

Comment:
it was technically possible to have no `target` here (which was surfaced by a test with changed setup) if you have a syntethic event like `new Event('click')`.